---
title:      Mentoring Students
permalink:  /mentoring/

---
<hr>
In my current postdoctoral work at the University of Maryland I've proposed, developed and managed DNA barcoding flow for our lab research: I've set up DNA extraction, PCR, and electrophoresis protocols, proposed and acquired reagents and equipment, set up sequencing software, managed sequencing service, trained and coordinated lab members. This DNA barcoding work became the main part of several ongoing projects in our lab, such as: wetland-stream connectivity; dragonflies diet composition; and host plant usage of potato leafhoppers and the spotted lanternfly. 

Over the past 2 years, I've mentored 10 students in various DNA barcoding research projects. I trained the students in tissue preparation, DNA isolation, PCR, visualization results using gel electrophoresis, sequence analysis and depositing the sequences to the NCBI GenBank datase, as well as retrieving data from published studies as part of our planned systematic review on molecular approaches to study insect diet. All this work has resulted (so far) in 10 student GenBank submissions and 2 student school symposium presentations (with 1 award). I also have one manuscript in preparation with one of my mentees as a coauthors. I enjoy working with my students and seeing their progress, and I'm very happy for their big and small successes.



<hr>

## Ecology of species interactions: use of molecular diet analysis 
<hr>

I mentored two graduate students and several undergraduate students in their research projects which involved insect diet analysis using various molecular biology techniques. The students worked on plant DNA detection from the potato leafhopper and the spotted lanternfly guts: we tried to identify insect host plants, confirm their ingestion, and explore insect host plant switch during insect development. The students isolated plant DNA sequences, conducted sequence analysis, and learn how annotate the sequence and deposit it to GenBank.


<h3><strong>DNA barcoding protocols we use:</strong></h3>

* <a href="{{ 'assets/content/misc/DNAbarcoding_in_pictures.pdf' | relative_url }}">Overview of the main DNA barcoding steps</a> 
* <a href="{{ 'assets/content/misc/DNAextraction_protocol.pdf' | relative_url }}">DNA extraction</a>
* <a href="{{ 'assets/content/misc/Primer_dilutions.pdf' | relative_url }}">Primer dilutions</a> 
* PCR protocols: (a) <a href="{{ 'assets/content/misc/PCRprotocol_isopods.pdf' | relative_url }}">isopods</a>; (b) <a href="{{ 'assets/content/misc/PCRprotocol_leafhoppers_lanternfly.pdf' | relative_url }}">leafhoppers</a>; (c) <a href="{{ 'assets/content/misc/PCRprotocol_leafhoppers_lanternfly.pdf' | relative_url }}">spotted lanternfly</a>
* <a href="{{ 'assets/content/misc/PCRpurification_protocol_ExoSAP-IT.pdf' | relative_url }}">PCR product purification</a> 
* <a href="{{ 'assets/content/misc/GelElectrophoresis_protocol.pdf' | relative_url }}">Gel electrophoresis</a>
* Sanger sequencing: <a href="{{ 'assets/content/misc/SequenceAnalysis_BioEdit_steps.pdf' | relative_url }}">sequence editing and alignment</a>
* Next-generation sequencing: (a) <a href="{{ 'assets/content/misc/NGS_sample_prep_submit.pdf' | relative_url }}">sample preparation and submission</a>; (b) <a href="{{ 'assets/content/misc/RetrievingNGSresults_steps.pdf' | relative_url }}">retrieving sequences from GENEWIZ</a>

<div class="col-sm">
    <figure class="text-center">
    <img class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/misc/pcr.jpg' | relative_url }}" alt="">
    <figcaption class="figure-caption">This is a drawing of our PCR protocol which one of my mentees made while I was explaining the main steps. I was very impressed! (Sep, 2019)
    </figcaption>
    </figure>
</div>

<hr>
<h3><strong>1. Molecular gut content analysis of the potato leafhopper</strong></h3>

<strong>Bryan</strong>, a high school student, was working with us during last Spring semester, and I was really impressed by his incredible progress during those several months. Bryan joined us in January knowing nothing about wet lab techinques for DNA barcoding process; he worked once a week only learning all the steps; and by the end of April he had successfully extracted genomic DNA from 100+ leafhopper samples, amplified plant DNA from leafhopper guts and succesfully obtained and deposited four plant sequences. His work was the essential part of our paper on the use of molecular markers for plant DNA detection from leafhopper gut contents (the manuscript is currently in preparation). 

<div class="row">
  <div class="col-sm">
    <figure class="text-center">
    <img class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/misc/img-11.jpg' | relative_url }}" alt="">
    <figcaption class="figure-caption">Setting up feeding trials with the potato leafhopper and first plant DNA amplification from the leafhopper gut contents (Jan-Mar, 2019)
    </figcaption>
    </figure>
  </div>
  <div class="col-sm">
    <figure class="text-center">
    <img class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/misc/bryan_1.jpg' | relative_url }}" alt="">
    <figcaption class="figure-caption">The results of Bryan's successfull work: four plant sequences, obtained and deposited to the NCBI GenBank database (May-June, 2019)
    </figcaption>
    </figure>
  </div>
  </div>



<h4>Updates:</h4>

* 09/23/19: I was very pleased to learn recently that Bryan was admitted to the University of Maryland where he continues studying Molecular Biology. Bryan was also featured in [The John Carroll School News](https://www.facebook.com/johncarrollschool/photos/a.549607655156223/2147147948735511/?type=3&theater). I'm so happy for him!

<hr>
<h3><strong>2. Molecular gut content analysis of the spotted lanternfly</strong></h3>
Over the past 2 years, I've been mentoring several students in different aspects of the molecular gut content analysis of the spotted lanternfly. <strong>Jessica</strong> and <strong>Nurani</strong>, our former lab managers, conducted DNA extractions and PCR amplification of a portion of trnL-intron from ingested plants; <strong>Darsy</strong>, a Ph.D. student, helped with obtaining and depositing plant sequences to the NCBI GenBank database; <strong>Hannah</strong>, an undergraduate student, is currently working on meta-barcoding of the lanternfly gut contents using a NGS-approach. 

<strong>Hannah</strong>, my current mentee and a great helper, is making an incredible progress: after less than a month of training she is now independently conducting all the steps from DNA isolation, PCR, and preparing samples for NGS sequncing. Hannah is currently working on identification of obtained sequences, as well as a qualitative analysis of the spectrum of ingested host plants. She is also actively participating in another project, a systematic review of molecular approaches to decipher insect herbivore diet (manuscript is in preparation with Hannah as a coauthor).


<div class="row">
  <div class="col-sm">
    <figure class="text-center">
    <img class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/misc/hannah_1.jpg' | relative_url }}" alt="">
    <figcaption class="figure-caption">Hannah conducts DNA extractions and PCR amplification of the plant DNA regions obtained from the spotted lanternfly gut contents (Jan 15, 2020; College Park, MD) 
    </figcaption>
    </figure>
  </div>

<div class="col-sm">
    <figure class="text-center">
    <img class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/misc/hannah_2.jpg' | relative_url }}" alt="">
    <figcaption class="figure-caption">The spotted lanternfly nymphs ascending and descending a host tree at the collection site (July 20, 2018; Berks County, PA) 
    </figcaption>
    </figure>
  </div>

  <div class="col-sm">
    <figure class="text-center">
    <img class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/misc/hannah_3.jpg' | relative_url }}" alt="">
    <figcaption class="figure-caption">Fourth instar nymph of the spotted lanternfly, Lycorma delicatula (July 20, 2018; Berks County, PA) 
    </figcaption>
    </figure>
  </div>
</div>

<h4>Updates:</h4>

* 04/09/20: To date, my mentees (in bold) have succesfully submitted to GenBank a total of 10 sequences for the chloroplast trnL-gene. All of them have been approved and are already available in the NCBI GenBank database: 

1. <strong>Illahi, N.</strong>, Avanesyan, A. and W. O. Lamp. (2020) Lonicera maackii ribulose-1,5-bisphosphate carboxylase/oxygenase large subunit (rbcL) gene, partial cds; chloroplast. Direct Submission, GenBank Accession no. [MN631052](https://www.ncbi.nlm.nih.gov/nuccore/MN631052)
1. <strong>Smith, D.K.</strong>, Avanesyan, A. and W. O. Lamp. (2020) Acer platanoides tRNA-Leu (trnL) gene, partial sequence; chloroplast. Direct Submission, GenBank Accession no. [MN450067](https://www.ncbi.nlm.nih.gov/nuccore/MN450067)
1. <strong>Smith, D.K.</strong>, Avanesyan, A. and W. O. Lamp. (2020) Acer rubrum tRNA-Leu (trnL) gene, intron; chloroplast. Direct Submission, GenBank Accession no. [MN450068](https://www.ncbi.nlm.nih.gov/nuccore/MN450068)
1. <strong>Smith, D.K.</strong>, Avanesyan, A. and W. O. Lamp. (2019) Eupatorium serotinum tRNA-Leu (trnL) gene, partial sequence; chloroplast. Direct Submission, GenBank Accession no. [MN395725](https://www.ncbi.nlm.nih.gov/nuccore/MN395725)
2. <strong>Smith, D.K.</strong>, Avanesyan, A. and W. O. Lamp. (2019) Lonicera maackii tRNA-Leu (trnL) gene, partial sequence; chloroplast. Direct Submission, GenBank Accession no. [MN365276](https://www.ncbi.nlm.nih.gov/nuccore/MN365276)
3. <strong>Smith, D.K.</strong>, Avanesyan, A. and W. O. Lamp. (2019) Pisum sativum isolate slf-2 tRNA-Leu (trnL) gene, partial sequence; chloroplast. Direct Submission, GenBank Accession no. [MN335637](https://www.ncbi.nlm.nih.gov/nuccore/MN335637)
4. <strong>Stancliff, B.</strong>, Avanesyan, A. and W. Lamp. (2019) Vicia faba var. major tRNA-Leu (trnL) gene, partial sequence; chloroplast. Direct Submission, GenBank Accession no. [MK934667](https://www.ncbi.nlm.nih.gov/nuccore/MK934667.1) 
5. <strong>Stancliff, B.</strong>, <strong>Smith, D.</strong>, Avanesyan, A. and W. Lamp. (2019) Pisum sativum tRNA-Leu (trnL) gene, partial sequence; chloroplast. Direct Submission, GenBank Accession no. [MK919208](https://www.ncbi.nlm.nih.gov/nuccore/MK919208.1) 
6. <strong>Stancliff, B.</strong>, <strong>Abdelwahab, O.</strong>, Avanesyan, A. and W. Lamp. (2019) Vigna unguiculata tRNA- Leu (trnL) gene, partial sequence; chloroplast. Direct Submission, GenBank Accession no. [MK883492](https://www.ncbi.nlm.nih.gov/nuccore/MK883492.1) 
7. <strong>Stancliff, B.</strong>, <strong>Ho, J.</strong>, Avanesyan, A. and W. Lamp. (2019) Helianthus annuus tRNA-Leu (trnL) gene, partial sequence; chloroplast. Direct Submission, GenBank Accession no. [MK875279](https://www.ncbi.nlm.nih.gov/nuccore/MK875279.1)

Four of these sequences were obtained by my former mentee, <strong>Bryan</strong>, a high school student!
<hr>

## Wetland-stream connectivity: DNA barcoding of isopods 
<hr> 

'Wetland-stream connectivity' connectivity is yet another exciting project; my mentee, <strong>Nina</strong>, a high school student, was working on this project more than year. During her work she created a reference library of mitochondrial sequences from stream isopod species - she then 'BLASTed' sequences from wetland isopod individuals against this library to determine the matches and estimate how 'close' wetland species were to stream species. 

<div class="row">
  <div class="col-sm">
    <figure class="text-center">
    <img class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/projects/novel_associations/img-20.jpg' | relative_url }}" alt="">
    <figcaption class="figure-caption">Developing PCR protocol with a black fly sample (BF) as a control (Oct-Nov, 2018)
    </figcaption>
    </figure>
  </div>
  <div class="col-sm">
    <figure class="text-center">
    <img class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/misc/nina-3.jpg' | relative_url }}" alt="">
    <figcaption class="figure-caption">Nina's first successful PCR amplification of isopod mitochondrial COI gene (December 10, 2018)
    </figcaption>
    </figure>
  </div>
  </div>

<div class="row">
  <div class="col-sm">
    <figure class="text-center">
    <img class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/misc/nina001.jpg' | relative_url }}" alt="">
    <figcaption class="figure-caption">Nina presented her final poster at the ERHS Research Symposium (April 26, 2019; Greenbelt, MD) 
    </figcaption>
    </figure>
  </div>
</div>

<h4>Updates:</h4>

* 05/25/19: Nina was admitted to the University of Maryland where she will study Environmental Science and Policy. Very exciting! She is continuing, however, her work on DNA barcoding of isopods this summer, learning and improving the phylogenetic reconstructions for our isopod samples.

* 04/26/19: Nina succesfully presented the final results of her year-long project at her school's research symposium!

* 02/22/19: Nina presented her research project earlier this week at her school's science fair and got 3rd prize! So exciting! 



 
<hr>

## Microscopy
During my postdoctoral work at the University of Wisconsin-Madison I trained a high school student in tissue dissection and isolation, slide preparation, tissue and cell identification, morphological analysis. Claire did a research project on the spotted wing drosophila through the Biotechnology Youth Apprenticeship program. Claire learned how to use a new protocol I developed for dissecting fly tissues and determining fly mating status. She successfully applied this new technique to different fly specimens; she also created beautiful illustrations for an invited manuscript which we published later in Insects. Here are some of the figures Claire made for our <a href="https://www.ncbi.nlm.nih.gov/pubmed/28287438">paper</a>:

<div class="row">
  <div class="col-sm">
    <figure class="text-center">
    <img class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/misc/claire.jpg' | relative_url }}" alt="">
    <figcaption class="figure-caption">Scheme of Drosophila suzukii‘s reproductive system and a spermatheca of a mated Drosophila suzukii female under the compound microscope at 20× objective lens; the scale is 50 µm (Avanesyan et al., 2017; Images by Claire Mattmiller)
    </figcaption>
    </figure>
  </div>
</div>

Additionally, Claire compiled a large dataset for temperature and humidity data we used in <a href="https://academic.oup.com/ee/article-abstract/47/6/1365/5159616">another paper</a> on the effect of temperature and humidity on the seasonal phenology of the spotted wing drosophila. 

<hr>

## PlantingScience Mentor


As a [PlantingScience fellow](https://plantingscience.org/members/5520/profile), during summer/fall 2017 I participated in the [PlantingScience program](https://plantingscience.org/) (Botanical Society of America; Digging Deeper project). I was an online scientist mentor and liaison for student research teams (high school and middle school students).

That was my first year with the PlantingScience program. I really enjoyed our summer workshop in Colorado Springs and meeting with biology teachers from around the country, as well as our wonderful trip to the Garden of the Gods. Last Fall, I was a scientist mentor for nine research teams and a liaison for a biology teacher and three research teams in her class. The student teams which I mentored, focused on [The Power of Sunlight](https://plantingscience.org/posinformation) module and conducted their independent research projects on plant photosynthesis and respiration. 

Since then, I stayed with PlantingSceince, and I've been serving as a scientist mentor every semester. During Fall 2018, I worked as a member of Master Plant Science Team serving  as a liaison for one class and a mentor for three student teams. Great experience! I'm continously looking forward to being a part of this interesting program every semester!

<figure class="text-center">
  <img class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
       src="{{ '/assets/content/misc/plantingscience1.jpg' | relative_url }}" alt="">
       <figcaption class="figure-caption">Digging Deeper Summer Professional Development<br>
    July 23-27, 2017. Colorado Springs, CO.
  </figcaption>
</figure>

<figure class="text-center"> 
  <img alt="" 
    class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/misc/plantingscience2-s.jpg' | relative_url }}" alt="">
  <img 
    class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/misc/plantingscience3-s.jpg' | relative_url }}" alt="">
  <figcaption class="figure-caption">A wonderful field trip to the Garden of the Gods. July 26, 2017. Colorado Springs, CO.</figcaption>
</figure>

<hr>

## Animal Ecology
In 2003, at Herzen State University, I started teaching a course on Animal Ecology. Since then, I’ve supervised the research of
several undergraduate students, including short projects (as a part of their independent research),
as well as their final theses (a Diploma in Russian academia) <a href="{{ site.baseurl }}{% link pages/courses/animal-ecology-students.md %}">...more
information</a>
{% assign path="/assets/content/courses/animalecology/studentwork/" %}

{% for i in (0..1) %}
  <div class="row mt-4 mb-4">
  {% for j in (1..3) %}
    {%- capture id -%}{{ i | times: 3 | plus: j }}{%- endcapture -%}
    <div class="col-12 col-md col-sm-6">
      <figure class="figure mb-0">
      <img id="mtrigger-{{ id }}" class="ic4f-mtrig ic4f-zoomin figure-img img-fluid" src="{{ site.baseurl }}{{ path }}tiger{{ id }}-s.jpg" alt="">
      </figure>
    </div>
  {% endfor %}
  </div>
{% endfor %}
Photo credit: Alexandr Mogilev. Thesis project: Ecological adaptations of the Felidae family: example of the Siberian tiger, Pantera tigris altaica (Leningrad Zoo, St. Petersburg, Russia, 2005)

