---
title:      Research Areas
permalink:  /research/
---
<div style="padding: 10px;width:350px;text-align:center;background: #e9f0e9;">
NEW: <strong><a href="{{ site.baseurl }}{% link pages/projects/research-protocols.md %}">research protocols</a></strong> I've developed
</div>

<hr>

In my research, I explore various mechanisms underlying ecology and evolution of species interactions using molecular, morphological, and behavioral approaches. I’m using arthropods and plant-insect study systems to invesitgate eco-evolutionary mechanisms which drive species interactions, and especially novel interactions between native and introduced species. I’m working on a variety of projects that involve field and greenhouse experiments, molecular biology techniques, light and scanning electron microscopy, and morphometric analysis.

I've just joined [Dr. Paula Shrewsbury's lab](https://shrewsburylab.weebly.com/) (at the Department of Entomology, University of Maryland) as an Assistant Research Scientist. In my new position, I continue my research on novel species interactions and I will focus on native and introduced parasitoid interactions for biocontrol of the invasive brown marmorated stink bug. This is very exciting and I'm looking forward to my new research adventure!

Below are my research projects from my postdoctoral work in [Dr. Bill Lamp’s lab](http://www.clfs.umd.edu/entm/lamp/index.html) (also at the Department of Entomology at UMD), as well as previous research projects; and here are the <a href="{{ site.baseurl }}{% link pages/projects/research-protocols.md %}">research protocols</a> which I've developed.

<hr>
<h2><strong>Hidden diversity of trophic interactions: use of molecular gut content analysis</strong></h2>
<hr>
<h4><strong>1. Novel plant-herbivore interactions: can plant DNA be detected within gut contents of sap-feeders? </strong></h4>

Plant DNA detection in insect herbviore gut contents is one of the most accurate ways to confirm host plant
utilization, determine insect diet and interactions with other organisms. Most of the previous
studies that involved molecular analysis of insect gut contents were primarily conducted on
leaf-chewing insects, such as beetles, moths, or grasshoppers. Sap-feeders could be more challenging
for molecular analysis of their gut contents because phloem sap presumably doesn’t contain plant
DNA. However, this method was shown to be effective for potato psyllids (Cooper et al 2016) -
  apparently, while feeding an insect stylet (a piercing mouthpart) can consume not only phloem sap
but can also pick up some of the surrounding plant cells which allows DNA to be detected in the
insect gut contents. 

In this project, I'm using two sap-feeding insects - the potato leafhopper and the spotted lanternfly - to investigate their host plant range. So far, I'm using two plant DNA barcoding regions - trnL and rbcL, as well as Sanger sequencing. I successfully isolated and identified plant DNA for 'single-DNA'-samples (i.e. when there is only one ingested host plant species in the insect gut), and we are planning to use NGS technology to explore mixed DNA samples and compose the insect host plant range.

<div class="row">
  <div class="col-sm">
    <figure class="text-center">
    <img class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/projects/novel_associations/img-12.jpg' | relative_url }}" alt="">
    <figcaption class="figure-caption">The spotted lanternfly, Lycorma delicatula; 4th instars (PA, 2018)
    </figcaption>
    </figure>
  </div>
  <div class="col-sm">
    <figure class="text-center">
    <img class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/projects/novel_associations/img-13.jpg' | relative_url }}" alt="">
    <figcaption class="figure-caption">The spotted lanternfly, Lycorma delicatula; 4th instars (PA, 2018)
    </figcaption>
    </figure>
  </div>
  <div class="col-sm">
    <figure class="text-center">
    <img class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/misc/a_1.jpg' | relative_url }}" alt="">
    <figcaption class="figure-caption">Preparing DNA extractions of L. delicatula gut contents
    </figcaption>
    </figure>
  </div>
  </div>

<div class="row">
  <div class="col-sm">
    <figure class="text-center">
    <img class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/projects/novel_associations/img-21.jpg' | relative_url }}" alt="">
    <figcaption class="figure-caption">Experimental set up and detection of DNA of consumed fava beans in the guts of potato leafhoppers. Additionally, we've found that contamination of the insect surface with plant DNA is not significant
    </figcaption>
    </figure>
  </div>
  <div class="col-sm">
    <figure class="text-center">
    <img class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/projects/novel_associations/img-22.jpg' | relative_url }}" alt="">
    <figcaption class="figure-caption">First amplification of the chloroplast trnL gene from the lanternfly body ('head+mouthparts' and 'gut content'-samples) 
    </figcaption>
    </figure>
  </div>
</div>

<h4>Updates:</h4>

* 04/01/20: Our [paper](https://doi.org/10.3390/insects11040215) on molecular gut content analysis of the invasive spotted lanternfly, Lycorma delicatula, was published today in Insects special issue ["Molecular Gut Content Analysis: Deciphering Trophic Interactions of Insects”](https://www.mdpi.com/journal/insects/special_issues/gut_trophic_interact)! So exciting!

* 02/14/20: The manuscript entitled "Molecular gut content analysis reveals the host plant range of the invasive spotted lanternfly, Lycorma delicatula" has been submitted to Insects, [Special Issue "Molecular Gut Content Analysis: Deciphering Trophic Interactions of Insects"](https://www.mdpi.com/journal/insects/special_issues/gut_trophic_interact)

* 01/09/20: we've submitted an abstract for our oral presentation (Lamp W.O and A.Avanesyan "Molecular gut content analysis reveals the host plant range of the invasive spotted lanternfly, Lycorma delicatula") to 2020 Joint Eastern and Southeastern ESA meeting in Atlanta, GA.

* 11/27/19: the results are ready, and the manuscript "Molecular gut content analysis reveals the host plant range of the invasive spotted lanternfly, Lycorma delicatula" is currently in preparation.

* 10/21/19: a grant proposal entitled "Identification of host plant use by the invasive spotted lanternfly (Lycorma delicatula) using next-gen DNA sequencing technology" has been submitted to Maryland Agricultural Experiment Station Competitive Grant Program.

* 10/01/19: I'm a Guest Editor for journal Insects, [Special Issue "Molecular Gut Content Analysis: Deciphering Trophic Interactions of Insects"](https://www.mdpi.com/journal/insects/special_issues/gut_trophic_interact)

* 09/10/19: the manuscript has been submitted to Environmental Entomology.

* 11/15/18: presented! Here is my <a href="{{ 'assets/content/publications/2018_ESA_talk.pdf' | relative_url }}">presentation</a>.

* 07/07/18: we are making some good progress, and we've submitted an abstract for our oral presentation (A.Avanesyan and W. O. Lamp, "Use of molecular markers for plant DNA to determine host plant usage for potato leafhopper, Empoasca fabae") to the ESA meeting in Vancouver, BC, Canada. 

<h4>Representative publications and presentations:</h4>

1. <strong>Avanesyan, A.</strong>, and W.O. Lamp. (2020) Use of molecular gut content analysis to decipher the range of food plants of the invasive spotted lanternfly, Lycorma delicatula. Insects: Special Issue " Molecular Gut Content Analysis: Deciphering Trophic Interactions of Insects", 11(4), 215<a class="ic4f-nowrap" href="/assets/content/publications/2020_SLF_gut_content.pdf"><i class="material-icons md-18">file_download</i><span class="ic4f-download">full text</span></a>

1. <strong>Avanesyan, A.</strong>, and W. Lamp (2018) Use of molecular markers for plant DNA to determine host plant usage for potato leafhopper, Empoasca fabae. Annual Meeting of the Entomological Society of America: 2018 ESA, ESC, and ESBC Joint Annual Meeting, Vancouver, BC, Canada. Oral presentation. <a class="ic4f-nowrap" href="/assets/content/publications/2018_ESA_talk.pdf"><i class="material-icons md-18">file_download</i><span class="ic4f-download">full text</span></a>

11. <strong>Avanesyan, A.</strong>, and W.O. Lamp. Use of molecular markers for gut content analysis of potato leafhopper, Empoasca fabae. <em>(In prep.)</em>

<hr>
<h4><strong>2. Deciphering the host plant range of invasive species: eDNA meta-barcoding of insect gut contents</strong></h4>

We continue exploring host plant usage of the spotted lanternfly, an aggressive invader in the eastern US. In this study, we are using next-generation sequencing technology as a promising approach for meta-barcoding plant species from the gut contents of polyphagous insect herbivores. Our current findings of plant DNA detection from the gut contents of the lanternfly nymphs (Avanesyan and Lamp, manuscript is in preparation) suggest that while the observed lanternfly nymphs actively move on a host plant they may not utilize it for feeding. A substantial number of the nymphs (~90%) we have analyzed (collected during summer 2018) showed the ingested DNA from a host plant other than the plant from which the nymphs were collected. To reconstruct a list of all possible host plant species of the lanternfly we are starting conducting meta-barcoding of the lanternfly gut contents using NGS approach ('Amplicon-EZ').
Our primary objectives include (1) identifying host plant utilization at different nymphal stages of L. delicatula through detection of plant host DNA within insect gut contents, (2) determining the longevity of the plant DNA in the gut contents of L. delicatula at different nymphal stages, and, using these results, (3) investigating feeding behavior of different nymphal stages of L. delicatula on multiple host plants.

<div class="row">
  <div class="col-sm">
    <figure class="text-center">
    <img class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/projects/novel_associations/plan.jpg' | relative_url }}" alt="">
    <figcaption class="figure-caption">Overview of the proposed activities for the project
    </figcaption>
    </figure>
  </div>
  <div class="col-sm">
    <figure class="text-center">
    <img class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/projects/novel_associations/ngs.jpg' | relative_url }}" alt="">
    <figcaption class="figure-caption">BLAST results of the gut content of the 3rd nymphal instar collected from a maple tree 
    </figcaption>
    </figure>
  </div>
</div>

<h4>Updates:</h4>

* 01/03/20: our MAES grant proposal on using a NGS-approach for meta-barcoding the lanternfly gut contents has just been funded! 

* 10/21/19: a grant proposal entitled “Identification of host plant use by the invasive spotted lanternfly (Lycorma delicatula) using next-gen DNA sequencing technology” has been submitted to Maryland Agricultural Experiment Station Competitive Grant Program.
<hr>


<h4><strong>3. Prey-predator interactions: molecular identification of prey items in predator gut contents and feces</strong></h4>

This is a new lab project I'm involved in and it is also a new challenge in my DNA barcoding work. I amplify DNA from dragonfly prey items. In addition to a gut content analysis, I’m working with degraded DNA from dragonfly feces. So far, I successfully amplified and sequenced COI partial gene from dragonfly feces produced during the first 2 hours post ingestion. The sequence analysis with subsequent BLAST results showed that the prey item was a crane fly. Surprisingly enough the DNA from the crane fly wasn’t degraded and an isolated fragment of 667 bp showed good sequence quality. These exciting results can make identification of dragonfly prey possible and accurately confirm (or even point to new) trophic interactions in dragonfly natural habitats.

<div class="row">
  <div class="col-sm">
    <figure class="text-center">
    <img class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/misc/odonata.jpg' | relative_url }}" alt="">
    <figcaption class="figure-caption">Obtaining the DNA of a crane fly from dragonfly feces.
    </figcaption>
    </figure>
    </div>
</div>

<h4>Updates:</h4>

* 06/10/19: DNA from mayfly was detected in feces produced during first 7 hours post ingestion by the same dragonfly individual. The sequence had lower quality but it was still readable.

<hr>
<h4><strong>4. Molecular approaches to diet analysis of insect herbivores: a systematic review</strong></h4>

The idea for this project came from our DNA work with the lanternfly gut contents. I tried a few regions of the chloroplast DNA which were candidates for plant DNA detection within the lanternfly guts, and only one of them yielded good amplification and sequencing results. Unlike relatively straight forward DNA barcoding of 'fresh' plants the situation with ingested plants are more complex: the plant DNA is degraded, plant DNA concentration in insect guts is often extremely low (compared to that in the intact plants), and we often do not know the time of consumption before we extract the ingested plant DNA. Choosing and testing various loci are always time-consuming, and even though there are more and more studies being published on molecular gut content analysis, the information on how to choose the best targeted gene for detecting ingested plant DNA is very limited.

Together with Hannah, an undergraduate student, my great helper and mentee, we have retrieved 902 experimental studies on insect gut content analysis from 4 public databases and 3 research journals. We are currently screening these articles according our several inclusion criteria and selecting the ones we are going to proceed with to the data analysis. Ultimately, we are interested in the following two questions:
(1) what regions of plant DNA are commonly used for insect molecular gut content analysis?
(2) what validation methods do the researchers use to determine the success of detection of ingested plant DNA?

We hope that outcomes of our systematic review would help us provide the methodological guidelines for the researchers for choosing the targeted regions of plant DNA which can be reliably detected within insect gut contents.

<div class="row">
  <div class="col-sm">
    <figure class="text-center">
    <img class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/misc/system_review.jpg' | relative_url }}" alt="">
    <figcaption class="figure-caption">First resuls from the extracted data based on 39 studies and 63 experimental records which satisfied our inclusion criteria.
    </figcaption>
    </figure>
    </div>
</div>

<h4>Updates:</h4>

* 02/05/20: secondary references are retrieved from the selected articles; we are currently extracting data from the studies.

* 11/20/19: 902 research articles on insect gut content analysis published between 1977-2019 were retrieved; 105 of relevant titles and 68 relevant abstracts were screened; of those, 55 articles were selected for full-text screening.

<hr>

<h2><strong>Wetland-stream connectivity: DNA barcoding of native isopods as a tool for assessing watershed connectivity</strong></h2>
<hr>

This is an ongoing project in Dr. Lamp’s lab. Using isopods from multiple wetland sites and streams in Maryland Delmarva Bays Wetlands we explore the potential connectivity between wetland and stream communities. Morphological identification of isopod species is tricky, and DNA barcoding, which I’m focusing on, is very helpful for estimating how ‘close’ wetland isopod species are to the isopod species inhabiting streams. I mentored three students in this DNA barcoding work, and together with my mentees we isolated and sequenced a portion of mitochondrial gene, cytochrome oxidase 1 (CO1), created a reference library of these sequences, and using a BLAST search engine in the NCBI GenBank database we assessed species and genera identity of isopods.

To date, we were able to identify seven unique stream species and four unique wetland species of isopods. We didn't find an overlap between species which suggests presumably isolation of stream and wetland populations of isopods and may potentially provide an evidence that stream and wetland arthropod communities do not overlap as well. We are currently working on reconstruction of phylogenetic relationships between these isopod species.

<div class="row">
  <div class="col-sm">
    <figure class="text-center">
    <img class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/projects/novel_associations/img-18.jpg' | relative_url }}" alt="">
    <figcaption class="figure-caption">Maryland Delmarva Bays (May, 2018) 
    </figcaption>
    </figure>
  </div>
  <div class="col-sm">
    <figure class="text-center">
    <img class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/projects/novel_associations/img-19.jpg' | relative_url }}" alt="">
    <figcaption class="figure-caption">Collecting isopods 
    </figcaption>
    </figure>
  </div>
  </div>

<div class="row">
  <div class="col-sm">
    <figure class="text-center">
    <img class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/projects/novel_associations/img-20.jpg' | relative_url }}" alt="">
    <figcaption class="figure-caption">Developing PCR protocol with a black fly sample (BF) as a control
    </figcaption>
    </figure>
  </div>
  <div class="col-sm">
    <figure class="text-center">
    <img class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/misc/nina-3.jpg' | relative_url }}" alt="">
    <figcaption class="figure-caption">First successful amplification! 
    </figcaption>
    </figure>
  </div>
  <div class="col-sm">
    <figure class="text-center">
    <img class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/misc/nina-4.jpg' | relative_url }}" alt="">
    <figcaption class="figure-caption">Our current reconstruction of phylogenetic relationships between stream and wetland isopod species (December 23, 2019)
    </figcaption>
    </figure>
  </div>
  </div>
  

<h4>Updates:</h4>

* 10/06/19: We are currently exploring phylogenetic relationships between stream and wetland isopod species.

* 04/26/19: Nina presented her final poster at the ERHS Research Symposium. 

* 02/22/19: Nina, my mentee, a high school student, who was working on this project since September 2018, presented her research project earlier this week at her school’s science fair and got 3rd prize! So exciting!


<hr>

<h2><strong>Morphological adaptations of invasive insects to host plant usage: use of scanning electron microscopy</strong></h2>
<hr>

To additionally explore the morphological adaptations of the lanternfly to host plant utilization, I focus on the following two objectives: (a) to assess changes in morphology of the lanternfly mouthparts (stylets and labium), and (b) to assess changes in morphology of the lanternfly tarsal tips (arolia and tarsal claws) at each developmental stage. The labium, stylets, and tarsal tips are the structures which are associated with primary contact of the lanternfly with its host plant, and which potentially facilitate the lanternfly successful host plant use. I assess the developmental changes in these structures using both scanning electron microscopy (SEM) and morphometric analysis. We expect these structures to undergo substantial morphological and morphometric changes throughout the lanternfly development which could potentially indicates the lanternfly association with certain host trees at each developmental stage. 

<div class="row">
  <div class="col-sm">
    <figure class="text-center">
    <img class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/projects/novel_associations/img-15.jpg' | relative_url }}" alt="">
    <figcaption class="figure-caption">Tarsal tip of the 3rd instar: tarsal claws + arolium
    </figcaption>
    </figure>
  </div>
  <div class="col-sm">
    <figure class="text-center">
    <img class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/projects/novel_associations/img-16.jpg' | relative_url }}" alt="">
    <figcaption class="figure-caption">Labium tip of the 3rd instar with numerous sensilla
    </figcaption>
    </figure>
  </div>
  <div class="col-sm">
    <figure class="text-center">
    <img class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/projects/novel_associations/img-17.jpg' | relative_url }}" alt="">
    <figcaption class="figure-caption">Labrum and stylets of the 4th instar
    </figcaption>
    </figure>
  </div>
</div>

<h4>Updates:</h4>

* 12/27/19: Our paper "External morphology and developmental changes of tarsal tips and mouthparts of the invasive spotted lanternfly, Lycorma delicatula" has been just published in [PLOS ONE](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0226995)! Here is our <a href="{{ 'assets/content/publications/2019_SLFmorphology.pdf' | relative_url }}">paper</a>.

* 11/18/19: I've presented our updated results on the lanternfly external morphology at the ESA meeting. Here is <a href="{{ 'assets/content/publications/2019_ESA_poster.pdf' | relative_url }}">my poster</a>.  

* 11/14/19: I've obtained all the sequences from the lanternfly gut content samples we used and finished data analysis. We've got some interesting results, and I'm currently working on getting the paper done (Tentative title: "Molecular gut content analysis reveals the host plant range of the invasive spotted lanternfly, Lycorma delicatula"). We are planning to submit it to Insects, the special issue on molecular gut content analysis of insect herbivores. Bill and I are co-editing this [special issue](https://www.mdpi.com/journal/insects/special_issues/gut_trophic_interact) and we hope to see many interesting studies there.  

* 06/06/19: Our paper on external morphology of the spotted lanternfly has been submitted to PLOS ONE.

* 02/28/19: I've finished the 2nd round of scanning electron microscopy of the lanternfly mouthparts and tarsi. Most of the beautiful SEM images are included in <a href="{{ 'assets/content/publications/2019_EBESA_poster.pdf' | relative_url }}"> my poster</a> for the upcoming EB-ESA meeting.

* 02/23/19: I've presented at the [Annual Meeting of the Maryland Organic Food & Farming Association](https://marylandorganic.org/events/) (Maryland Dept. of Agriculture, Annapolis, MD). <a href="{{ 'assets/content/publications/2019_moffa.pdf' | relative_url }}">My talk</a> was on the lanternfly biology, behavior, and host plant usage.

* 01/25/19: I'm presenting my intermediate results at the at the Entomological Society of America Annual Meeting, Eastern Branch, March 9-11,2019. Blacksburg, VA.

<h4>Representative publications and presentations:</h4>

1. <strong>Avanesyan, A.</strong>, Maugel T.K., and W. Lamp (2019) External morphology and developmental changes of tarsal tips and mouthparts of the invasive spotted lanternfly, Lycorma delicatula. PLOS ONE, doi.org/10.1371/journal.pone.0226995 <a class="ic4f-nowrap" href="/assets/content/publications/2019_SLFmorphology.pdf"><i class="material-icons md-18">file_download</i><span class="ic4f-download">full text</span></a> 

1. <strong>Avanesyan, A.</strong>, Maugel, T., and W. Lamp. (2019) External morphology and developmental changes of tarsal tips and mouthparts of the invasive spotted lanternfly, Lycorma delicatula. Annual Meeting of the Entomological Society of America, St. Lois, MO. Poster presentation <a class="ic4f-nowrap" href="/assets/content/publications/2019_ESA_poster.pdf"><i class="material-icons md-18">file_download</i><span class="ic4f-download">full text</span></a>  

1. <strong>Avanesyan, A.</strong> (2019) Spotted lanternfly: information and update. Maryland Organic Food & Farming Association, Maryland Dept. of Agriculture, Annapolis, MD. Invited talk <a class="ic4f-nowrap" href="/assets/content/publications/2019_moffa.pdf"><i class="material-icons md-18">file_download</i><span class="ic4f-download">full text</span></a>  

<hr>

<h2><strong>Should I eat or should I go: a systematic review and meta-analysis</strong></h2>
<hr>

This was a systematic review I conducted last year. The review aimed to
identify patterns of grasshopper feeding preferences for native versus introduced plants and,
consequently, grasshopper potential for biotic resistance of native communities, that (as the review
has shown) is surprisingly overlooked in experimental studies on invasion ecology.


<div class="row">
  <div class="col-sm">
    <figure class="text-center">
    <img class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/projects/novel_associations/img-5.jpg' | relative_url }}" alt="">
    <figcaption class="figure-caption">My table pet for inspiration..
    </figcaption>
    </figure>
  </div>
  <div class="col-sm">
    <figure class="text-center">
    <img class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/projects/novel_associations/meta-analysis.jpg' | relative_url }}" alt="">
    <figcaption class="figure-caption">Literature search and data colelction: PRISMA flowchart
    </figcaption>
    </figure>
  </div>
</div>

<h4>Updates:</h4>

* 10/07/18: the <a href="{{ 'assets/content/publications/2018_should_i_eat.pdf' | relative_url }}">manuscript</a> has been published in [Plants](https://www.mdpi.com/2223-7747/7/4/83)! I was also featured in our departmental [news](https://entomology.umd.edu/news/alina-avanesyan-published-in-plants).

* 07/07/18: I've submitted a poster presentation on this review to the upcoming 
[ESA meeting in Vancouver, BC, Canada](https://www.entsoc.org/event-calendar/2018-esa-esc-and-esbc-joint-annual-meeting-entomology-2018). 
I'm also going to present it here, at the University of Maryland research
symposium organized by the Office of Postdoctoral Affairs.

* 05/18/18: the manuscript has been submitted!

<h4>Representative publications and presentations:</h4>

1. <strong>Avanesyan, A.</strong> (2018) Should I eat or should I go? Acridid grasshoppers and their novel host plants: implications for biotic resistance. Plants: Special Issue "Plants Interacting with other Organisms: Insects", 7(4), 83. Invited paper <a class="ic4f-nowrap" href="/assets/content/publications/2018_should_i_eat.pdf"><i class="material-icons md-18">file_download</i><span class="ic4f-download">full text</span></a>  

1. <strong>Avanesyan, A.</strong> (2018) Should I eat or should I go? Acridid grasshoppers and their novel host plants: implications for biotic resistance. Annual Meeting of the Entomological Society of America: 2018 ESA, ESC, and ESBC Joint Annual Meeting, Vancouver, BC, Canada. Poster presentation <a class="ic4f-nowrap" href="/assets/content/publications/2018_ESA_poster.pdf"><i class="material-icons md-18">file_download</i><span class="ic4f-download">full text</span></a>  

<hr>

<h2><strong>Herbivore resistance and tolerance in invasive Miscanthus sinensis cultivars</strong></h2>
<hr>
Although exotic chinese silver grass, a gorgeous ornamental plant and important biofuel source, can be highly invasive in some states, not all of its cultivars are invasive. For this project, I conducted field and greenhouse experiments to explore plant resistance and tolerance to grasshopper herbivory, how these responses differ among Miscanthus cultivars, and whether the initially introduced wild type demonstrates the highest level of herbivore resistance.

<div class="row">
  <div class="col-sm">
    <figure class="text-center">
    <img class="ic4f-mtrig ic4f-zoomin figure-img img-fluid ic4f-max-height-md"
    src="{{ '/assets/content/projects/novel_associations/img-10.jpg' | relative_url }}" alt="">
    <figcaption class="figure-caption">Miscanthus sinensis cultivars I use in my experiments (from left to right): ‘Dixieland’, ‘Autumn Anthem’,‘Zebrinus’,‘Morning Light’,‘Gracillimus’
    </figcaption>
    </figure>
  </div>
</div>

<h4>Updates:</h4>

* 09/20/19: I've finished the analysis of the data we collected in 2018 and 2019 summer seasons; we see some interesting patterns in plant tolerance to herbivory and their association with plant morphological appearance; and I'm about to start working on the manuscript. 


<hr>

<h2><strong>Reproductive biology and distribution of invasive agricultural pests (previous postdoctoral work)</strong></h2>
<hr>

In my postdoctoral work at the University of Wisconsin-Madison I studied one of the emerging insect pests – the spotted wing drosophila,
Drosophila suzukii Matsumura (Diptera: Drosophilidae). This is a highly invasive insect species,
which attacks undamaged ripening fruit of a wide variety of soft-skinned fruits and berries. Native
to Southeast Asia, currently it is observed in Europe, North America, and South America. Drosophila
suzukii has demonstrated a very high dispersal capacity and remarkable phenotypic plasticity –
during only a couple of decades since its first introduction in Hawaii, D. suzukii invaded different
temperate regions and now is being monitored in many northern and eastern states, as well as Canada.

I work on several projects on different aspects of D. suzukii biology and population distribution. I
coordinated a multi-state bait comparison project for determining optimal attractants for D.
suzukii. This project was conducted in Minnesota; we set up fly traps with eight different baits and
conducted monitoring of D. suzukii in raspberry during several weeks. I also developed experimental
design for the spatial and temporal distribution project which was supposed to be conducted later in
the season when population of D. suzukii could be established.

<img class="w-75 border-0 figure-img img-fluid mx-auto d-block" 
  src="/assets/content/projects/{{ page.project_id }}/steps.jpg" alt="">

{% include figure-group.html 
  hr=0
  img_path="/assets/content/projects/drosophila/"
  src1="spermateca1.jpg" alt1=""
  src2="spermateca2.jpg" alt1=""
%}


I was also actively involved in a project on D. suzukii seasonal phenology focused on overwintering
of D. suzukii and the effect of temperature and humidity on D. suzukii seasonal abundance. We are
currently writing a paper on D. suzukii seasonal phenology which includes an analysis of the
interactions between D. suzukii seasonal abundance and temperature and humidity dynamics during the
collecting seasons in 2014-2015.

Additionally, I developed a protocol for tissue preparation, isolating spermathecae, and determining
mating status of D. suzukii, which we have applied in our bait comparison and phenology studies.
This protocol has been recently published in Insects (Special issue on invasive species). In this
paper, we also demonstrated how this protocol can be applied for both field collected flies and
flies reared in the lab, including fly specimens stored on a long-term basis.

{% include figure-group.html 
  img_path="/assets/content/projects/drosophila/"
  src1="raspberry.jpg" alt1=""
  caption1="Summer and fall raspberry varieties at West Madison <br>Agricultural Research Station on May 19, 2016."
  src2="barrier.jpg" alt2=""
  caption2="Plastic high tunnel (A, B) with raspberry plants (C) at Hoch Orchard & Gardens farm (La Crescent, MN)"
%}

<h4>Representative publications:</h4>

1. Guédot, C., <strong>Avanesyan, A.</strong>, and K. Hietala-Henschell. (2018) Effect of temperature and humidity on the seasonal phenology of Drosophila suzukii (Diptera: Drosophilidae) in Wisconsin. Environmental Entomology 47(6): 1365–1375. <a class="ic4f-nowrap" href="/assets/content/publications/2018_SWD_phenology.pdf"><i class="material-icons md-18">file_download</i><span class="ic4f-download">full text</span></a>  

1. Jaffe, B.D., <strong>Avanesyan, A.</strong>, Bal, H. K., Grant, J., Grieshop, M.J., Lee, J.C., Liburd, O.E., Rhodes, E., Rodriguez-Saona, C., Sial, A.A., Zhang, A., and C. Guédot (2018) Multistate comparison of attractants and the impact of fruit development stage on trapping Drosophila suzukii (Diptera: Drosophilidae) in raspberry and blueberry. Environmental Entomology, 47(4): 935–945. <a class="ic4f-nowrap" href="https://academic.oup.com/ee/advance-article-abstract/doi/10.1093/ee/nvy052/4970567"><i class="material-icons md-18">exit_to_app</i><span class="ic4f-download">link to publication</span></a>

1. <strong>Avanesyan, A.</strong>, Jaffe, B.D., and C. Guédot (2017) Isolating spermatheca and determining mating status of the invasive spotted wing drosophila, Drosophila suzukii: a protocol for tissue dissection and its applications. Insects: Special issue “Invasive Insect Species”. 8(1), 32; doi:10.3390/insects8010032. Invited paper. <a class="ic4f-nowrap" href="/assets/content/publications/2017_isolating.pdf"><i class="material-icons md-18">file_download</i><span class="ic4f-download">full text</span></a>  

<hr>
<h2><strong>Novel plant-insect interactions (doctoral research)</strong></h2>
<hr>

In my dissertation at the University of Cincinnati I explored the interactions between insect
herbivores and their host plants within the context of invasion ecology. Specifically, I was
interested in the potential impact of generalist insects on the successful spread of exotic plants.
Using a grasses-grasshoppers model, I combined behavioral and molecular approaches to explore (1)
tolerant and resistant responses of native and exotic grasses to herbivory by grasshoppers, and (2)
grasshopper feeding preferences on these plants. I conducted laboratory and field experiments at two
research centers (University of Cincinnati and University of Maryland) to explore whether plant
responses to different insect populations were similar, and whether the insects acted in the same
way.

**Ph.D. Dissertation**: Native versus exotic Grasses: the interaction between generalist insect
herbivores and their host plants. (Alina Avanesyan, 2014; University of Cincinnati)

**Advisor**: <a href="http://homepages.uc.edu/~culleyt/CulleyLab.html">Dr. Theresa Culley</a>,
Professor, Department of Biological Sciences, University of Cincinnati

<h4><strong>(1) Plant DNA detection from grasshopper gut contents</strong></h4>

As part of my dissertation, I developed a new PCR-based method for accurate detection of plant meals from grasshopper guts, which had not been described in experimental studies. Using the developed protocol, I successfully amplified fragments (~500 bp) of the non-coding region of the chloroplast trnL (UAA) gene from grasshopper guts; the plant DNA was found to be detectable up to twelve hours post ingestion (PI) in nymphs and up to 22 h PI in adult grasshoppers. This <a href="{{ 'assets/content/publications/2014_dna.pdf' | relative_url }}">method</a> was published in Applications in Plant Sciences and was <a href="{{ 'assets/content/publications/2014_dna_pressrelease.pdf' | relative_url }}">featured</a> in Botanical Society of America News, ScienceDaily, ScienceNewsLine, Phys.org, LabRoots, EurekAlert!, as well as Down to Earth (a science
magazine of The Society For Environmental Communications in India).

<div class="mt-4 mb-4">
  {% include figure-group.html 
    hr=0
    img_path="/assets/content/projects/dissertation_phd/"
    src1="fig3.jpg" alt1=""
    src2="dissect.jpg" alt2=""
  %}
</div>
<div class="embed-responsive embed-responsive-16by9 mx-auto w-75">
  <iframe 
    class="d-block mx-auto embed-responsive-item"
    src="https://www.youtube.com/embed/cze938ABUw0" frameborder="0" allowfullscreen></iframe>
</div>

<br>
<h4><strong>(2) Plant resistance and tolerance to herbivory; grasshopper feeding preferences</strong></h4>

My study grasses species were: native Andropogon gerardii and Bouteloua curtipendula, and exotic,
potentially invasive, Miscanthus sinensis and Bothriochloa ischaemum. I conducted choice (many
grasses) and no-choice (one grass species) experiments; experiments in the field and in the
greenhouse; experiment with intact plants and with their clipped. To estimate plant resistance and
tolerance I measured the amount of leaf tissue consumed by grasshoppers, the growth of plants during
the feeding and their re-growth after the feeding, the grasshopper’s growth on different plants and
many other things. In the same behavioral experiments with plants and, additionally, in the experiments with clipped
leaves, I estimated leaf consumption, the proportion of leaves consumed by grasshoppers, as well as
their assimilation efficiency and relative consumption rate on native and exotic grasses. I was conducting most of my experiments in Ohio (UC Center for Field Studies and The Culley Laboratory), but also conducted some at the Western Maryland Research & Education Center
– thanks to Dr. William Lamp who was a member of my dissertation committee and provided me with
access to this wonderful research facility, and Tim Ellis, the Center’s Agronomy Program Manager,
whose help with preparing the plot and growing plants was invaluable.

<div class="mt-4 mb-4">
  {% include figure-group.html 
    hr=0
    img_path="/assets/content/projects/dissertation_phd/"
    src1="fig1.jpg" alt1=""
    src2="field.jpg" alt2=""
  %}
</div>

  Also see: <a href="{{ site.baseurl }}{% post_url 2013-01-08-how-to-build-a-cage %}">How to build a cage</a>


My results from both, behavioral experiments and molecular confirmation of diet, demonstrated that
exotic grasses have lower resistance (the ability to reduce damage) to grasshopper feeding than
native grasses; whereas plant tolerance (the ability to maintain fitness while sustaining damage) to
herbivory does not differ between native and exotic grasses. My results suggested that exotic
grasses that do not have a coevolutionary history with native grasshoppers are less adapted to
reduce damage from these herbivores, although they tolerate them similar to native plants. These
results contributed to our understanding of the trade-off between plant tolerance and plant
resistance to herbivory and the possible mechanisms of the success of exotic plans in the introduced
range – i.e., mechanisms that facilitate plant invasion. The important applications of this project
are: effective control of invasive plants, predictions of plant invasion, and ecological restoration
of native communities. 

<h4>Representative publications:</h4>

1. <strong>Avanesyan, A.</strong>, and T.M. Culley (2016) Tolerance of native and exotic prairie grasses to herbivory by Melanoplus grasshoppers: application of a non-destructive method for estimating plant biomass changes as a response to herbivory. The Journal of the Torrey Botanical Society 144(1):15-25. <a class="ic4f-nowrap" href="http://www.bioone.org/doi/abs/10.3159/TORREY-D-15-00028"><i class="material-icons md-18">exit_to_app</i><span class="ic4f-download">link to publication</span></a>

1. <strong>Avanesyan, A.</strong>, and T.M. Culley (2015) Feeding preferences of Melanoplus femurrubrum grasshoppers on native and exotic grasses: behavioral and molecular approaches. Entomologia Experimentalis et Applicata 157: 153-163. <a class="ic4f-nowrap" href="http://onlinelibrary.wiley.com/doi/10.1111/eea.12356/abstract"><i class="material-icons md-18">exit_to_app</i><span class="ic4f-download">link to publication</span></a>

1. <strong>Avanesyan, A.</strong>, and T.M. Culley (2015) Herbivory of native and exotic North-American prairie grasses by nymph Melanoplus grasshoppers. Plant Ecology 216: 451-464. <a class="ic4f-nowrap" href="/assets/content/publications/2015_herbivory.pdf"><i class="material-icons md-18">file_download</i><span class="ic4f-download">full text</span></a>

1. <strong>Avanesyan, A.</strong> (2014) Plant DNA detection from grasshopper gut contents: a step-by-step protocol, from tissues preparation to obtaining plant DNA sequences. Applications in Plant Sciences 2(2):1300082. <a class="ic4f-nowrap" href="/assets/content/publications/2014_dna.pdf"><i class="material-icons md-18">file_download</i><span class="ic4f-download">full text</span></a>

<hr>

<h2><strong>Microsatellite motif study</strong></h2>   
<hr>

During my doctoral studies at Cincinnati, I was also involved in microsatellite motif study
conducted in Dr. Theresa Culley’s lab. Microsatellites, also known as simple sequence repeats (SSR)
or short tandem repeats (STR), are typically defined as repeated sequences of 1-6 bases found
throughout the nuclear and plastid genomes of eukaryotes. For many researchers, microsatellites
continue to be the marker of choice for surveys of genetic diversity and structure, as well as
paternity analysis and mating system estimates in which codominance is essential.

In this project we explored the relationship between microsatellite motif type and detectable
genetic variation in plant populations, which is critical when a researcher determines which markers
are associated with higher levels of genetic variation. To guide researchers in their choice of
molecular markers, we conducted a literature review based on 6,616 microsatellite markers published
from 1997-2012. We examined relationships between heterozygosity (He or Ho) and allele number (A)
with the following marker characteristics: repeat type, motif length, repeat frequency, and
microsatellite size, as well as their variations across taxa. Our results showed no significant
differences in genetic variation between imperfect and perfect repeat types, but dinucleotide motif
lengths exhibited significantly higher A, He, and Ho than most other motifs. Repeat frequency was
positively correlated with A, He, and Ho, but correlations with microsatellite size were minimal or
non-significant. We concluded that researchers should carefully consider their choices specific to
the desired application; and if researchers aim to find high genetic variation, dinucleotide motif
lengths with large repeat frequencies are recommended.  

<h4>Representative publication:</h4>

* Merritt, B.J., Culley, T.M., <strong>Avanesyan, A.</strong>, Stokes, R., and J. Brzyski (2015) An empirical review: Characteristics of plant microsatellite markers that confer greater levels of genetic variation. Applications in Plant Sciences 3(8):1500025. <a class="ic4f-nowrap" href="/assets/content/publications/2015_microsatellite.pdf"><i class="material-icons md-18">file_download</i><span class="ic4f-download">full text</span></a>

<hr>

<h2><strong>Phylogenetic relationships of the Sarcophagidae (Diptera)</strong></h2>
<hr>
I conducted this project in Dr. Ron DeBry’s lab at the beginning of my doctoral program at the
University of Cincinnati. Flies of the Sarcophagidae family are known as forensically important
insects; that is why studies of their identification and systematic relationships are critically
important. The phylogenetic analysis which included mtDNA (COI, COII and ND4) provided good
resolution for most nodes. However, the mtDNA gene tree might not be identical to the species tree,
so it was necessary to obtain additional data from nuclear genes. Such new data might increase the
support for the mtDNA tree or support a different phylogeny. Our results showed that adding nuclear
PER gene in most cases maintained or improved support in the Bayesian Inference tree. However, we
also find support for some novel relationships.

As a side project, I also worked on host-parasite interaction between sarcophagid flies and
grasshoppers. Grasshoppers cause significant damage to crops and rangeland which leads to economic
losses in the US and worldwide. Chemical insecticides are traditionally used to prevent
grasshoppers’ outbreaks but these chemicals are harmful to the ecosystem and are costly for pest
managers. Parasitic flesh flies are one promising approach to biological control of grasshoppers.
Fly larvae have a noticeable impact on reproductive physiology and survival of grasshoppers. This
topic had potential applications for biological control of grasshoppers. I went on two collection
trips to Montana and Iowa and maintained grasshoppers in the lab to observe the presence of
parasitoids. However, I did not detect any, which might be explained by a relatively low natural
infection rate (<3%). I continued to work with current literature, which contained controversial
information about this host-parasite model and I tried to figure out what species of flies can
infect what species of grasshoppers based on North-American experimental studies.

<h4>Representative presentation:</h4>

* <strong>Avanesyan A.</strong>, Stamper T.I. , Timm A., Wong E., Dahlem G.A., and R. DeBry (2010) Phylogenetic relationships of the Sarcophagidae (Diptera), using three mitochondrial loci (COI, COII, and ND4) and one nuclear locus (PER). 58th Annual Meeting of the Entomological Society of America. San Diego, CA. Poster presentation <a class="ic4f-nowrap" href="/assets/content/publications/2010_phylogenetic.pdf"><i class="material-icons md-18">file_download</i><span class="ic4f-download">full text</span></a>

<hr>

<h2><strong>Genetic variation in littoral snail populations</strong></h2>
<hr>

I worked on this project while I was a part-time researcher at the <a
href="http://www.cytspb.rssi.ru/">Institute of Cytology, RAS</a> in St.  Petersburg, Russia. I
worked at the Laboratory of Cell Biology in Culture under the direction of <a
href="https://www.researchgate.net/profile/Natalia_Mikhailova">Dr.  Natalia Mikhailova</a>. The
project was on molecular phylogeny and ecological adaptations of the Cerastoderma and the Littorina
snails collected in the intertidal zone of North-European region. We have recently published our
paper on the evidence of hybridization between two species of littoral snails in the intertidal
sites of the Barents Sea, using RAPD nuclear marker.

<h4>Representative publication:</h4>

* Granovitch, А.I., Maximovich, A.N., <strong>Avanesyan, A.V.</strong>, Starunova Z.I., and N.A. Mikhailova (2013) Micro-spatial distribution of two sibling periwinkle species across the intertidal indicates hybridization. Genetica 141 (7): 293-301. <a class="ic4f-nowrap" href="/assets/content/publications/2013_microspatial.pdf"><i class="material-icons md-18">file_download</i><span class="ic4f-download">full text</span></a>

<hr>

<h2><strong>Host-parasite interactions: host immune response to infection (dissertation research)</strong></h2>
<hr>

**Dissertation. Candidate of Science:** <a href="{{ 'assets/content/misc/AlinaAvanesyan_dissertation_2002.pdf' | relative_url }}">The effect of defense responses of snails on
development of trematode partenitae (with a focus on the family Echinostomatidae)</a> (Alina
Avanesyan, 2002; Herzen State University, St. Petersburg, Russia)

**Advisor:** Dr. Gennady Ataev, Professor, Department Chair, Department of Zoology, Herzen State
University, St. Petersburg, Russia

The focus of my Candidate of Science dissertation was cellular immune response of Biomphalaria
snails to infection by Echinostoma trematodes. Biomphalaria snails are freshwater pulmonate snails,
native to Caribbean and South America. In my research, I used two species, B. glabrata and B.
pfeifferi. Biomphalaria glabrata has been a primary model species for investigating snail defense
mechanisms because it is an intermediate host for the human blood fluke, Schistosoma mansoni, a
dangerous parasite that infects millions of people worldwide and causes the disease that is called
schistosomiasis. For my research, I used other, less dangerous, but still wide-spread trematodes –
Echinostoma caproni and E. paraensei, which are intestinal parasites of mammals and birds and which
also undergo their larvae development in Biomphalaria snails. In previous studies on B. glabrata,
two strains of this species have been identified: a strain which exhibits resistance to infection by
E. caproni (my primary study species) and a strain which is susceptible to infection by this
trematode. In my research, I was interested in exploring the role of hemocytes in snail defense
mechanisms and how a hemocytic response might differ between resistant and susceptible snails.

I worked under the direction of <a href="https://www.researchgate.net/profile/Gennady_Ataev">Dr.
Gennady Ataev</a> in the <a
href="https://www.herzen.spb.ru/main/structure/fukultets/bio/1384520856/1387808264/">Laboratory of
Experimental Zoology</a> at the <a
href="https://sites.google.com/a/zooherzen.org/www/home">Department of Zoology</a> at <a
href="https://www.herzen.spb.ru/en/">Herzen State University</a>. Snail defense mechanisms to
infection by trematodes was the main focus of the lab research. For my dissertation project, my
primary objectives were (1) to identify and characterize the hematopoietic tissue in B. glabrata
snails, (2) to compare the structure, mitotic activity of the hematopoietic tissue, as well as
encapsulation of parasites by hemocytes in resistant and susceptible strains (both qualitatively and
quantitatively), and (3) to perform morphological analysis of Echinostoma larvae (sporocyst)
development in B.  glabrata snails.

<h4><strong>(1) Location and morphology of the hematopoietic tissue</strong></h4>

By the time I started this project, the location of the hematopoietic tissue in invertebrates, and
particularly in snails, was poorly understood. We found that the hematopoietic tissue in
Biomphalaria snails (so called amebocyte-producing organ, or APO) is located between the pericardium
and the mantle epithelium. APO is composed of numerous cellular aggregations; they are located near
the external surface of the pericardium and lacunas of the blood system. The number of cell
aggregations In non-infected snails did not exceed 5; the cross-sectional area of each aggregation
ranged from 25-50 µm. The aggregations varied from round, oval, and elongated to amoeboid shape, and
comprised of clustered cells with the basophilic cytoplasm and oval-shaped nuclei. 

<figure>
  <div class="row">
    <div class="col-12 col-sm">
      <figure class="figure mb-0">
        <img class="figure-img img-fluid" 
          src="{{ '/assets/content/projects/dissertation_cand/apo_a.jpg' | relative_url }}" alt="">
      </figure>
    </div>
    <div class="col-12 col-sm">
      <figure class="figure mb-0">
        <img class="figure-img img-fluid" 
          src="{{ '/assets/content/projects/dissertation_cand/apo_b.jpg' | relative_url }}" alt="">
      </figure>
    </div>
  </div>
  <div class="row">
    <div class="col-12 col-sm">
      <figure class="figure mb-0">
        <img class="figure-img img-fluid" 
          src="{{ '/assets/content/projects/dissertation_cand/apo_c.jpg' | relative_url }}" alt="">
      </figure>
    </div>
    <div class="col-12 col-sm">
      <figure class="figure mb-0">
        <img class="figure-img img-fluid" 
          src="{{ '/assets/content/projects/dissertation_cand/apo_d.jpg' | relative_url }}" alt="">
      </figure>
    </div>
  </div>
  <figcaption class="figure-caption">Histological sections of the amoebocyte-producing organ (APO) in resistant
  Biomphalaria glabrata snails. (A) APO (black arrows) in non-infected snails; (mc) mantle cavity;
  (pc) pericardial cavity. (B) APO with divided cells (dc) (white arrows) after 3 days post
  infection (PI). (C) APO after 3 days PI. (D) APO after 5 days PI.
  </figcaption>
</figure>

<h4><strong>(2) Hematopoietic activity in resistant and susceptible snails</strong></h4>

To further explore defense mechanisms B. glabrata snails, we performed a comparative histological
analysis of the structure and mitotic activity of the hematopoietic tissue, as well as migration of
hemocytes from the APO to the location of E. caproni sporocyst in resistant and susceptible snails.
Snails were dissected in certain time intervals (from 1h to 30d post infection); tissues were then
fixed and embedded in paraffin. Sections (5 µm thick) were prepared using a microtome, stained
(using Erlich’s hematoxylin-eosin), and screened under the microscope. We found that there was a
greater cellular immune response in resistant snails compared to that in susceptible snails – i.e.
increased cell proliferation in the APO, increased migration of hemocytes to the location of the
sporocyst, and successful encapsulation of the sporocyst. In susceptible snails, however, even
though we also observed increased cell proliferation in the APO and aggregations of hemocytes near
the sporocyst, this cellular response had never resulted in the encapsulation of the parasite, and
the sporocyst continued its development and migration to the snail heart area.

<figure class="figure">
  <img class="border-0 figure-img img-fluid"
       src="{{ '/assets/content/projects/dissertation_cand/capsules.jpg' | relative_url }}" alt="">
  <figcaption class="figure-caption text-left">Encapsulation of Echinostoma caproni mother
    sporocysts developing in resistant Biomphalaria glabrata snails (from 3 to 7 days post infection).
    The sporocysts demonstrate different level of degradation, and sporocysts remains only are
    observed after 7 days post infection (the last picture).
  </figcaption>
</figure>

<figure class="figure">
  <img class="border-0 figure-img img-fluid"
       src="{{ '/assets/content/projects/dissertation_cand/cells2.jpg' | relative_url }}" alt="">
  <figcaption class="figure-caption text-left">Activity of the amoebocyte-producing organ (APO)
     during infection by Echinostoma caproni in resistant (A) and susceptible (B) Biomphalaria
     glabrata snails. 
  </figcaption>
</figure>

<h4><strong>(3) Germinal cell lineage in Echinostoma miracidia</strong></h4>

As part of my dissertation, I explored development of Echinostoma trematodes in Biomphalaria snails.
I focused on the germinal elements of Echinostoma miracidia and performed comparative morphological
analysis of germinal cells development in E. caproni and E. paraensei. Germinal material in
Echinostoma miracidia is represented by germinal cells (primary and/or secondary) and
undifferentiated cells. The germinal cells are quite large (cross-sectional area is 27.0 ± 0.8 µm2),
with a large bubble-shaped nucleus and a basophilic cytoplasm. We found that miracidia of E.
paraensei already contained embryos, and their sporocysts released mother rediae a few days earlier
than sporocysts of E. caproni. Overall, our analysis of germinal cells development provided
additional support to previous findings that species E. caproni and E. paraensei distinct species.

<figure class="mx-auto w-75">
  <img class="border-0 figure-img img-fluid"
       src="{{ '/assets/content/projects/dissertation_cand/miracidia.jpg' | relative_url }}" alt="">
  <figcaption class="figure-caption text-left">Longitudinal sections of miracidia of Echinostoma caproni (A) and Echinostoma paraensei (B,C) (e,
    embryo of mother redia; gc, germinal cells; n, neural mass; sc, secretory cells; uc,
    undifferentiated cells). Large arrow indicates miracidium without a redial embryo. Scale bar, 10 pm.
  </figcaption>
</figure>

<h4>Representative publications and presentations:</h4>

1. Ataev, G.L., Dobrovolskij, A.A., <strong>Avanessian, A.V.</strong>, and E.S., Loker (2001) Germinal elements and their development in Echinostoma caproni and Echinostoma paraensei (Trematoda) miracidia. The Journal of Parasitology 87 (5): 1160-1164. <a class="ic4f-nowrap" href="/assets/content/publications/2001_germinal.pdf"><i class="material-icons md-18">file_download</i><span class="ic4f-download">full text</span></a>

1. Ataev, G.L., <strong>Avanessian, A.V.</strong>, Loker, E.S., and A.A. Dobrovolskij (2001) The organization of germinal material and dynamics of mother sporocyst reproduction in the genus Echinostoma (Trematoda: Echinostomatidae). Parazitologia 35 (4): 307-319.(In Russian) <a class="ic4f-nowrap" href="http://www.unboundmedicine.com/medline/citation/11605455/%5BThe_organization_of_germinal_material_and_dynamics_of_mother_sporocyst_reproduction_in_the_genus_Echinostoma__Trematoda:_Echinostomatidae_%5D"><i class="material-icons md-18">exit_to_app</i><span class="ic4f-download">link to publication</span></a> <a class="ic4f-nowrap" href="/assets/content/publications/2001_organization_russian.pdf"><i class="material-icons md-18">file_download</i><span class="ic4f-download">full text</span></a>

1. Ataev, G.L., A.A. Dobrovolskij, <strong>A.V. Avanessian</strong> and C. Coustau (2000) Significance of amoebocyte-producing organ of Biomphalaria glabrata snails (strains selected for susceptibility/resistance) in cellular response to Echinostoma caproni mother sporocysts infection. Proceedings of the symposium on ecological parasitology at the turn of the millennium, St. Petersburg, Russia, 1-7 July, 2000. Bulletin of the Scandinavian Society for Parasitology 10 (2): 65. <a class="ic4f-nowrap" href="/assets/content/publications/2000_significance.pdf"><i class="material-icons md-18">file_download</i><span class="ic4f-download">full text</span></a>





